src/entities/offerings.ts

```diff
@@ -71,6 +76,56 @@ export interface Price {
   readonly formattedPrice: string;
 }
 
+/**
+ * Represents the price and duration information for a phase of the purchase option.
+ * @public
+ */
+export interface PurchaseOptionPhase {
+  /**
+   * The duration of the purchase option price in ISO 8601 format.
+   * For applicable options (trials, initial/promotional prices), otherwise null
+   */
+  readonly periodDuration: string | null;
+  /**
+   * The price for the purchase option.
+   * Null in case of trials.
+   */
+  readonly price: Price | null;
+  /**
+   * The number of cycles this option's price repeats.
+   * I.e. 2 subscription cycles, 0 if not applicable.
+   */
+  readonly cycleCount: number;
+}
+
+/**
+ * Represents a possible option to purchase a product.
+ * @public
+ */
+export interface PurchaseOption {
+  /**
+   * The unique id for a purchase option
+   */
+  readonly id: string;
+}
+
+/**
+ * Represents a possible option to purchase a subscription product.
+ * @public
+ */
+export interface SubscriptionPurchaseOption extends PurchaseOption {
+  /**
+   * The base phase for a SubscriptionPurchaseOption, represents
+   * the price that the customer will be charged after all the discounts have
+   * been consumed and the period at which it will renew.
+   */
+  readonly basePhase: PurchaseOptionPhase;
+  /**
+   * The trial information for this subscription option if available.
+   */
+  readonly trialPhase: PurchaseOptionPhase | null;
```

Comment:
do these two need the "Phase" in the name? or can they just be "base" and "trial"

Reply:
Hmm I think it's fine to remove that yeah 👍